src/envo/envo-edit.owl

```diff
@@ -40171,6 +39843,535 @@ AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3400002> "ar
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_3510001> <http://orcid.org/0000-0002-7463-6306>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_3510001> "2020-08-03T17:32:09Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3510001> "brake fluid"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3510001> <http://purl.obolibrary.org/obo/ENVO_3510004>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_3510002> (automatic transmission fluid)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Automatic_transmission_fluid") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_3510002> "Fluid used for valve operation, brake band friction, and gear lubrication within an automatic vehicle transmission.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_3510002> <http://orcid.org/0000-0002-7463-6306>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_3510002> "2020-08-03T17:34:16Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_3510002> "ATF")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3510002> "automatic transmission fluid"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3510002> <http://purl.obolibrary.org/obo/ENVO_00003074>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_3510004> (hydraulic fluid)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Hydraulic_fluid") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_3510004> "A medium by which power is transferred in hydraulic machinery.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_3510004> <http://orcid.org/0000-0002-7463-6306>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_3510004> "2020-08-03T17:35:45Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_3510004> "http://purl.enanomapper.org/onto/ENM_8000199")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_3510004> "Common hydraulic fluids are based on mineral oil or water.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_3510004> "hydraulic fluid"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_3510004> <http://purl.obolibrary.org/obo/ENVO_00003074>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_3510005> (motor oil)
```

Comment:
As above, this should have a more informative superclass ([oil](http://purl.obolibrary.org/obo/ENVO_00002985)?) and use subclass axioms to be inferred into manufactured product.